### DISCUSSION TITLE
Unknown parameters value

### DISCUSSION THREAD

**Discussion by gigli69** [BobanSpasic/PSS-Revive#14] [Ideas]
As the goal of your software is to find hidden features, maybe add a toggle-button to switch all the "not used" fields directly =1 or =0 ?
So we could directly heard if there is something happening ?

**Comment by BobanSpasic** [comment]
I didn't have a lot of free time to play with it, but till now I couldn't find any effect of the "not used" parameters.
It can be, that the SysEx format for the PSS is shared with some other series, where the other series uses these "not used" parameters. Perhaps we got the crippled version of the chip.
Other possibility would be - the sound chip in PSS has 4 output/channels. It could be that not all the outputs are wired to the audio output, and that the rest of the parameters have something to do with the unused outputs. I don't know...
I was just curious, because the original voices have some of the "not used" parameters set.
Setting all to 1, or all to 0 didn't make any audible change in my tests, but to be honest - I didn't play much with it.
Another thing that caught my attention, is the SysEx voice format of Yamaha DSR-1000. There are a lot of similarities (DSR is 4-OP, PSS is 2-OP). Just take a look here:
https://www.mattmontag.com/development/unlocking-the-yamaha-dsr-2000
Both series are using just the low-nibble in SysEx for data. I think the FB-01 is also using that format.

Another test I wanted to do, is to see if there are differences with these "not used" parameters between PSS-480 and PSS-780 (I own these two now), but I didn't have luck.
As I've got to my grandma's house, where my old PSS-780 waited for me the last 20 years, I couldn't get it to work anymore. The most of the keyboard is working, but none of the buttons are working anymore. I didn't have enough time to repair it... Perhaps in summer, when I visit my grandma again.
My brother-in-law has a PSS-680. I could test on that one too in summer.
